terraform/060-simplesamlphp/vars.tf

```diff
@@ -1,3 +1,8 @@
+variable "secret_salt" {
+  default = ""
+  description = "This allows for porting the value over from a primary to a secondary workspace (a random string that is 64 characters long)."
```

Comment:
Looks like `terraform fmt` complained about the misaligned `=` signs, if you want to fix that.